minimint/src/consensus/interconnect.rs

```diff
@@ -7,11 +7,11 @@ use rand::CryptoRng;
 use secp256k1_zkp::rand::RngCore;
 use serde_json::Value;
 
-pub struct FediMintInterconnect<'a, R: RngCore + CryptoRng> {
-    pub fedimint: &'a FediMintConsensus<R>,
+pub struct MinimintInterconnect<'a, R: RngCore + CryptoRng> {
+    pub fedimint: &'a MinimintConsensus<R>,
```

Comment:
I like the name change and think we should do more of this. For example, the instance is called `fedimint` here and `mint_consensus` elsewhere. I get really confused when the same thing has multiple names. Not suggesting you change in this PR, but just commenting generally.

Reply:
Ah you are right I will change all occurrences of "fedimint"